students/jstrauss123/lesson03/mailroom.py

```diff
@@ -1,72 +1,101 @@
 #!/usr/bin/env python3
 
 # function - send thank you
-def fun_sendty():
+def send_thankyou(donorl):
+    print("")
+    # create donor_name_list
+    donor_name_list = [k for k,v in donorl]
+    # prompt for donor name
     full_name = input("Please provide full name of donor: ")
+    print("")
     while full_name == "list":
-        print(donor_list)
+        # print donor names
+        #print(donor_name_list)
+        for i,j in donorl:
+            print(i)
```

Comment:
```
for donors in donor_list:
    print(donors[0])
```
or 
```
for donor_name, _ in donor_list:
    print(donor_name)
```
it is best to be more explicit with var names and use `_` for unused vars if need to

Reply:
why not use the `donor_name_list`?

Reply:
fixed as directed, using _ for unused var, thanks for the guidance